collector/collector/config.py

```diff
@@ -32,32 +32,12 @@ class Config(object):
     # The maximum retention time in days for records stored in the database
     # which do not match the filters in PURGE_FILTERED_RECORDS.
     # Use 0 to avoid deletion of all unfiltered records.
-    MAX_DAYS_KEEP_UNFILTERED_RECORDS = 30
-    # A dictionary in the following format:
-    # {
-    #     "<field_name>": {
-    #         "<field_value>": <max_days_to_keep>,
-    #         ...
-    #         },
-    #     ...
-    # }
-    # Currently supported fields to filter:
-    # [
-    #     'severity',
-    #     'classification',
-    #     'machine_id'
-    # ]
-    # Use 0 to avoid deletion of records that matches the filter.
-    # If you do not want to filter records to delete, just set an empty dict '{}'
+    MAX_DAYS_KEEP_UNFILTERED_RECORDS = 35
+    # See config_example.py for details about PURGE_FILTERED_RECORDS
     PURGE_FILTERED_RECORDS = {
-        "severity": {
-            1: 5,
-            4: 0
-        },
         "classification": {
-            "org.clearlinux/mce/*": 0,
             "org.clearlinux/hello/world": 1,
-            "org.clearlinux/heartbeat/ping": 1,
+            "org.clearlinux/heartbeat/ping": 7,
```

Comment:
I think we should remove this filter by default, because the Population view depends on various time ranges for queries (currently: past 1 day, 3 days, 7 days, 14 days, and all days (default 35)).

There is an example of this filter in config_example.py, and that's probably good enough.

Reply:
Done :)